core/experiments/manager.php

```diff
@@ -529,12 +529,12 @@ private function render_feature_settings_field( array $feature ) {
 	}
 
 	private function render_feature_dependency( $feature ) {
-		if ( ! empty( $feature['dependency'] ) ) {
+		if ( ! empty( $feature['dependencies'] ) ) {
 			?>
 			<div class="e-experiment__dependency">
-				<b class="e-experiment__dependency__title"><?php echo esc_html__( 'Depends on:', 'elementor' ); ?></b>
-			<?php foreach ( $feature['dependency'] as $dependency ) : ?>
-				<span class="e-experiment__dependency__item"><?php echo $dependency->get_name(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?></span>
+				<strong class="e-experiment__dependency__title"><?php echo esc_html__( 'Depends on:', 'elementor' ); ?></strong>
```

Comment:
Move the `:` out of the translate

```suggestion
				<strong class="e-experiment__dependency__title"><?php echo esc_html__( 'Depends on', 'elementor' ); ?>:</strong>
```